.circleci/config.yml

```
@@ -34,7 +34,31 @@ jobs:
       - deploy:
           name: Deploy
           command: |
+
+            CURRYEAR="$(date +%Y)"
+            CURRDATE="$(date +%m/%d/%Y)"
+
+            # Holiday dates specific to 2021 that will need to be updated next year
+            HOLIDAYS=("03/12/2021" "04/19/2021" "05/28/2021" "05/31/2021" "07/02/2021" "07/05/2021"
+            "09/06/2021" "10/11/2021" "11/11/2021" "11/25/2021" "11/26/2021" "12/23/2021")
+
+            # List of Holidays the Broad is closed for every year
+            HOLIDAYSEACHYEAR=("01/01" "12/24" "12/25" "12/26" "12/27" "12/28" "12/29" "12/30" "12/31")
+
+            for val in "${HOLIDAYSEACHYEAR[@]}"; do
+               HOLIDAYS+=("${val}/${CURRYEAR}")
+            done
+
+            # Do not release on Broad Holidays
+            for val in "${HOLIDAYS[@]}"; do
+                 if [ ${val} = "${CURRDATE}" ]; then
```

Per `shellcheck` (https://www.shellcheck.net/): `SC2086: Double quote to prevent globbing and word splitting.`
```suggestion
                 if [ "${val}" = "${CURRDATE}" ]; then
```

---

Thanks so much for all your feedback @cjllanwarne !